src/System.Drawing.Common/tests/RegionTests.cs

```diff
@@ -0,0 +1,1930 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
```

Comment:
@karelz the question is not whether to have both, it's whether to remove this line `// The .NET Foundation licenses this file to you under the MIT license.`

@AlexGhiondea since you had the original thread with legal, could you please ask them whether we should have that line if there's non-Mono content in there? (Presumably pretty much all the files we borrowed from Mono will eventually have non-Mono content in there)

I don't think that should hold up checking this in - either way around :)